docs/source/api.rst

```diff
@@ -0,0 +1,212 @@
+  -H 'unstructured-api-key: <YOUR API KEY>' \
+  -F 'files=@sample-docs/family-day.eml' \
+  -F 'output_format="text/csv"'
+
+
+PDF Table Extraction
+=====================
+
+To extract the table structure from PDF files using the ``hi_res`` strategy, ensure that the ``pdf_infer_table_structure`` parameter is set to ``true``. This setting includes the table's text content in the response. By default, this parameter is set to ``false`` to avoid the expensive reading process.
+
+.. code:: shell
+
+  curl -X 'POST' \
+  'https://api.unstructured.io/general/v0/general' \
+  -H 'accept: application/json' \
+  -H 'Content-Type: multipart/form-data' \
+  -H 'unstructured-api-key: <YOUR API KEY>' \
+  -F 'files=@sample-docs/layout-parser-paper.pdf' \
+  -F 'strategy=hi_res' \
+  -F 'pdf_infer_table_structure=true' \
+  | jq -C . | less -R
+
+
+Strategies
+===========
+
+Four strategies are available for processing PDF/Images files: ``hi_res``, ``fast``, ``ocr_only``, and ``auto``. ``fast`` is the default ``strategy`` and works well for documents that do not have text embedded in images.
+
+On the other hand, ``hi_res`` is the better choice for PDFs that may have text within embedded images, or for achieving greater precision of `element types <https://unstructured-io.github.io/unstructured/getting_started.html#document-elements>`_ in the response JSON. Be aware that ``hi_res`` requests may take 20 times longer to process compared to the ``fast`` option. See the example below for making a ``hi_res`` request.
+
+.. code:: shell
+	
+  curl -X 'POST' \
+  'https://api.unstructured.io/general/v0/general' \
+  -H 'accept: application/json' \
+  -H 'Content-Type: multipart/form-data' \
+  -H 'unstructured-api-key: <YOUR API KEY>' \
+  -F 'files=@sample-docs/layout-parser-paper.pdf' \
+  -F 'strategy=hi_res' \
+  | jq -C . | less -R
+
+The ``ocr_only`` strategy runs the document through Tesseract for OCR. Currently, ``hi_res`` has difficulty ordering elements for documents with multiple columns. If you have a document with multiple columns that do not have extractable text, it's recommended that you use the ``ocr_only`` strategy. Please be aware that ``ocr_only`` will fall back to another strategy if Tesseract is not available.
+
+For the best of all worlds, ``auto`` will determine when a page can be extracted using ``fast`` or ``ocr_only`` mode, otherwise, it will fall back to hi_res.
+
+
+XML Tags
+=========
+
+When processing XML documents, set the ``xml_keep_tags`` parameter to ``true`` to retain the XML tags in the output. If not specified, it will simply extract the text from within the tags.
+
+.. code:: shell
+	
+  curl -X 'POST' \
+  'https://api.unstructured.io/general/v0/general' \
+  -H 'accept: application/json'  \
+  -H 'Content-Type: multipart/form-data' \
+  -H 'unstructured-api-key: <YOUR API KEY>' \
+  -F 'files=@sample-docs/fake-xml.xml' \
+  -F 'xml_keep_tags=true' \
+  | jq -C . | less -R
+
+
+Using the API Locally
```

Comment:
To me the preferred way to use the API locally is through the Docker container since that requires less manual set up. Can we move this section down and update the title to "Developing with the API Locally" or similar?